rust/src/transformations/make_stable_expr/expr_sum/mod.rs

```diff
@@ -49,9 +49,30 @@ where
 
     let (by, input_margin) = middle_domain.context.grouping("sum")?;
 
+    if middle_domain.column.nullable {
+        return fallible!(
+            MakeTransformation,
+            "input data might contain nulls. Preprocess your data with `.fill_null`."
```

Comment:
Should the column name be included in the message? It might not be obvious downstream.